### ISSUE TITLE
Problems cross-compiling Linux -> Windows

### ISSUE THREAD

**Issue by JalonSolov** [vlang/v#7169]
```
OS: linux, Ubuntu 20.04.1 LTS (WSL 2)
Processor: 32 cpus, 64bit, little endian, AMD Ryzen Threadripper 1950X 16-Core Processor
CC version: cc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0

getwd: /home/jalon/git/v/examples/tetris
vmodules: /home/jalon/.vmodules
vroot: /home/jalon/git/v
vexe: /home/jalon/git/v/v
vexe mtime: 2020-12-06 16:14:23
is vroot writable: true
is vmodules writable: true
V full version: V 0.1.30 434fa70.bac6be2

Git version: git version 2.25.1
Git vroot status: weekly.2020.49.5-61-gbac6be22
.git/config present: true
thirdparty/tcc status: thirdparty-linux-amd64 7543de81
```
**What did you do?**

`v -cc gcc -os windows .` in examples/tetris subdir

**What did you expect to see?**

`tetris.exe` compiled and ready to run on Windows.
 
**What did you see instead?**
```
jalon@Ripper:~/git/v/examples/tetris$ v -cc gcc -os windows .
/home/jalon/git/v/thirdparty/stb_image/stbi.o not found, building it in /home/jalon/.vmodules/cache/d9/d91f92913225b88dd944635206f52483.o ...

Cross compiling for Windows...
/usr/bin/x86_64-w64-mingw32-ld: /tmp/ccEk2hEX.o:tetris.2937154263795622410.tmp.c:(.text+0x5a5c5): undefined reference to `stbi_load'
/usr/bin/x86_64-w64-mingw32-ld: /tmp/ccEk2hEX.o:tetris.2937154263795622410.tmp.c:(.text+0x5a7fb): undefined reference to `stbi_load_from_memory'
/usr/bin/x86_64-w64-mingw32-ld: /tmp/ccEk2hEX.o:tetris.2937154263795622410.tmp.c:(.text+0x5a984): undefined reference to `stbi_image_free'
/usr/bin/x86_64-w64-mingw32-ld: /tmp/ccEk2hEX.o:tetris.2937154263795622410.tmp.c:(.text+0x5a9a3): undefined reference to `stbi_set_flip_vertically_on_load'
collect2: error: ld returned 1 exit status
Cross compilation for Windows failed. Make sure you have mingw-w64 installed.
Try `sudo apt install -y mingw-w64` on Debian based distros, or `sudo pacman -S mingw-w64-gcc` on Arch, etc...
jalon@Ripper:~/git/v/examples/tetris$
```

**Comment by JalonSolov** [comment]
In case it makes a difference...
```
jalon@Ripper:~$ /usr/bin/x86_64-w64-mingw32-gcc --version
x86_64-w64-mingw32-gcc (GCC) 9.3-win32 20200320
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

jalon@Ripper:~$
```

**Comment by spytheman** [comment]
Should be fixed by 4b4733d .

**Comment by spytheman** [comment]
![image](https://user-images.githubusercontent.com/26967/101388195-48bea380-38c8-11eb-9ba5-a56fd309db70.png)
